lib/rack/mock_response.rb

```diff
@@ -100,7 +128,7 @@ def parse_cookies_from_header
     def identify_cookie_attributes(cookie_filling)
       cookie_bits = cookie_filling.split(';')
       cookie_attributes = Hash.new
-      cookie_attributes.store('value', cookie_bits[0].strip)
+      cookie_attributes.store('value', Array(cookie_bits[0].strip))
```

Comment:
Do you mind explaining why this line was changed?

Reply:
`CGI::Cookie` does this in its constructor (it's also a subclass of `Array`). It's required for the wrapper so `value[0]` doesn't just return a single char.